libs/sdk-ui-web-components/README.md

```diff
@@ -0,0 +1,130 @@
+
+To integrate the library into your app, all you need to do is to integrate a script tag with a correct URL to the `<head>`
+section of your application.
+
+```html
+<script type="module" src="https://{your-GD-server-URL}/components/{workspaceId}.js?auth=sso"></script>
+
+<!-- for example -->
+<script type="module" src="https://analytics.example.com/components/my-workspace.js?auth=sso"></script>
+```
+
+Script has to be of type `module`, as we are using JavaScript modules for this distribution.
+
+An `auth` query parameter is optional. When set to `sso`, the library will trigger automatic SSO flow when user is not
+authenticated with GoodData server, i.e. will redirect user to the authentication flow and then back once user is authorized.
+
+## Custom authentication flow
+
+If you want to customize the authentication flow, you'll need to provide the authenticated backend by yourself. For example:
+
+```html
+<script type="module">
+    import { setContext } from "https://analytics.example.com/components/my-workspace.js";
+    import tigerFactory, {
+        ContextDeferredAuthProvider,
+        redirectToTigerAuthentication,
+    } from "https://analytics.example.com/components/tigerBackend.js";
+
+    setContext({
+        backend: tigerFactory()
+            .onHostname("https://analytics.example.com")
+            // Following is the part that you might want to change in case of a custom authentication flow
+            .withAuthentication(new ContextDeferredAuthProvider(redirectToTigerAuthentication)),
+        workspaceId: "my-workspace",
+    });
+</script>
+```
+
+## Embedding
+
+Once authentication is set and ready, you can embed Dashboard or a single visualization as follows:
+
+```html
+<!-- Embedding dashboard with ID "my-dashboard-id" -->
+<gd-dashboard dashboard="my-dashboard-id"></gd-dashboard>
+
+<!-- Embedding insight with ID "my-insight-id" -->
+<gd-insight dashboard="my-insight-id"></gd-insight>
+```
+
+### Dashboard Custom Element
+
+`gd-dashboard` supports the following attributes:
+
+-   `dashboard` - an ID of the dashboard to embed.
+-   `workspace` - optional, an ID of the workspace for this dashboard. By default, it's taken from the context (e.g. from the script URL).
+-   `readonly` - if enabled, the dashboard will be embedded in read-only mode disabling any user interaction that would alter any backend state (disabling creating/changing alerts, creating scheduled emails, and so on).
+-   `locale` - the localization of the visualization. For available languages, see [the full list of available localizations](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui/src/base/localization/Locale.ts).
+-   `mapbox` - the map access token to be used by [geo pushpin charts](https://sdk.gooddata.com/gooddata-ui/docs/geo_pushpin_chart_component.html#geo-config).
+
+`gd-dashboard` emits events from [this list](https://sdk.gooddata.com/gooddata-ui-apidocs/v8.9.0/docs/sdk-ui-dashboard.dashboardeventtype.html).
```

Comment:
Please use https://sdk.gooddata.com/gooddata-ui-apidocs/docs/sdk-ui-dashboard.dashboardeventtype.html so that it always points to the latest version.